css/src/components/gradient-card.scss

```diff
@@ -0,0 +1,11 @@
+.gradient-card {
```

Comment:
Note to self and others: should it become necessary in the future to support a card with a full gradient bg, this is a good step away. We could do that with the following, which isn't much of change from this:

```
	&:not(&:has(.gradient-card-content)) {
		color: $gradient-vivid-invert; // ⭐
		outline: $gradient-vivid-invert; // ⭐
	}
```